data-esnext-accepted.js

```diff
@@ -0,0 +1,1243 @@
+// exports browsers and tests
+// new browsers should only be added once they have at least a single 'true' result.
+exports.name = 'ES Next Accepted';
```

Comment:
I think a more accurate name here would be `ES 2017+` or `Finished ECMAScript Proposals`, per https://tc39.github.io/process-document/ - "Accepted" isn't a term that's usually used.